src/Components/TabsItem/TabsItem.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import "./TabsItem.scss";
+import Icons from "../Icons/Icons";
+
+function TabsItem({ title, content, icon }) {
+  const [clicked, setClicked] = useState(false);
```

Comment:
Could be a better naming.
```suggestion
  const [isOpen, setIsOpen] = useState(false);
```